src/main/java/rs/teslaris/core/converter/commontypes/GeoLocationDTOToGeoLocation.java

```diff
@@ -0,0 +1,15 @@
+package rs.teslaris.core.converter.commontypes;
+
+import rs.teslaris.core.dto.commontypes.GeoLocationDTO;
+import rs.teslaris.core.model.commontypes.GeoLocation;
+
+public class GeoLocationDTOToGeoLocation {
+    public static GeoLocation fromDTO(GeoLocationDTO geoLocationDTO) {
```

Comment:
We should make GeoLocationConverter which contains both this and toDTO logic.

Reply:
+1

Reply:
I created converter classes for every entity converter that we had. Also, I put Deprecated annotation in old ToDTO and DtoTo classes just in case it was used in other branches